lib/utils/args.js

```diff
@@ -54,21 +54,24 @@ function _normaliseFlagsAndArs(argsIntoCommand, cmd) {
     // bail on old style commands
     return argsIntoCommand;
   }
-  var processArgs = _.clone(process.argv);
+
+  const normalisedArgs = _.clone(argsIntoCommand);
+  
   _.each(cmd.alias, function(alias, key) {
     if (key.match(/[0-9]/)) {
       var position = parseInt(key, 10),
         arg = argsIntoCommand[position];
       var hasArgumentAtPosition = !isNaN(position) && typeof arg !== 'undefined' && !arg.match(/^--/) && arg.indexOf('=') === -1;
       if (hasArgumentAtPosition) {
-        var idx = processArgs.indexOf(arg);
+        var idx = normalisedArgs.indexOf(arg);
         if (idx > -1) {
-          processArgs[idx] = '--' + alias + '=' + arg;
+          normalisedArgs[idx] = '--' + alias + '=' + arg;
         }
       }
     }
   });
-  return processArgs;
+  
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)